### PR TITLE
Reached to Convergence

### LINKED ISSUE #5: Reach to Convergence
**Describe the enhancement**
The training process must reach to *convergence*.  **Convergence** is when the training process reaches a stable state. **Nesterov Momentum** is a technique that helps to speed up convergence.

## Resources
- https://medium.com/@ompramod9921/convergence-in-deep-learning-f96568923d43
- https://medium.com/@giorgio.martinez1926/nesterov-momentum-explained-with-examples-in-tensorflow-and-pytorch-4673dbf21998
- https://machinelearningmastery.com/how-to-reduce-overfitting-in-deep-learning-with-weight-regularization/
- https://towardsdatascience.com/this-thing-called-weight-decay-a7cd4bcfccab
- https://medium.com/analytics-vidhya/deep-learning-basics-weight-decay-3c68eb4344e9

### PR DESCRIPTION
**Pull request by sagedemage**
Did the best I can for the training process to reach to *convergence*. Set the optimizer's weight decay to 0.0001.

## Extra Tasks Done
1. Calculate the portion size for the training and validation sets. 
2. Corrected the image file paths for some of the `Image_1` and `Image_2` columns of the `labels/oxford-iiit-pet_labels.csv` file.

### CODE REVIEW

**Review by sagedemage — COMMENTED** [review]
LGTM!
